docs: Add SSI EG whitepaper on wallet concepts

marcelruland opened the pull request:
## Description

This PR adds a white paper as written by the SSI Expert Group on future wallet concepts in the Catena-X dataspace.

Note: Original title was "Enable Issuance Flow for Multiple Wallet Providers and Bring Your Own Wallet". Here changed to "Enable Issuance Flow for Multiple Wallet Providers and Bring Your Own Wallet".

## Pre-review checks

- [ ] ~DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.~ (not applicable)
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

matbmoser commented on docs/white-papers/0001-ssi-eg-wallet-concepts.md:
```diff
@@ -0,0 +1,796 @@
+## Useful links
+
+* Related concept: [Moving to DID-based Identifiers in the Catena-X Protocol Layer](https://github.com/catenax-eV/cx-ex-dataspace-connectivity/blob/main/docs/bpn-did-resolution/moving-to-DID-identifiers.md#moving-to-did-based-identifiers-in-the-catena-x-protocol-layer)
+* Related issue: Deprecation StatusList2021, BitStringStatusList current W3C standard
```
Also missing the License, example:
```
## NOTICE


This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).


- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2024, 2025 Contributors to the Eclipse Foundation
- Source URL: https://github.com/eclipse-tractusx/sig-architecture
```

marcelruland replied:
See license at the beginning of the file. Also see [your comment here](https://github.com/eclipse-tractusx/sig-architecture/pull/10#discussion_r2000851851).

matbmoser reviewed (changes requested):
please change the things and add the licenses

matbmoser commented:
I would question the naming convention for white papers with the ID, maybe keep the media inside of the whitepaper folder, and keep it as a readme. 

![image](https://github.com/user-attachments/assets/7996f44a-a141-466b-a3d1-02b0a64acf65)


please follow the same way we are documenting patterns

lgblaumeiser commented on docs/white-papers/0001-ssi-eg-wallet-concepts/README.md:
```diff
@@ -0,0 +1,798 @@
+
+### What happens with existing EDC contracts when the DID is changed?
+
+No implications. The move of the DID from one wallet to another one should only be seen as a key revocation.
```
Are you sure, the issue is, that the contract is based on an identifier, which is at least related to the DID, could be the BPN, but that is to be discussed.

lgblaumeiser commented on docs/white-papers/0001-ssi-eg-wallet-concepts/README.md:
```diff
@@ -0,0 +1,798 @@
+
+### Can a member company move from one holder wallet to another?
+
+This should pose no problem, except when changing the DID.
```
This answer I do not understand, what does that move mean if the DID does not change?

lgblaumeiser reviewed (changes requested):
I am willing to accept what is written, but some precision would support the paper.

evegufy commented:
@lgblaumeiser thanks for the feedback. the idea of this PR was to make the concept publicly available out of the SSI expert group to allow a public discussion. To allow a proper feedback cycle with the expert group, I recommend to open a discussion to collect the feedback once the PR has been merged.

matbmoser reviewed (approved):
Fine for me! @marcelruland @evegufy

matbmoser reviewed (changes requested):
I did not saw it.
please do not name the whitepaper as ssi eg wallet concepts!

Please remove the EG from the tractus-x things, also do not need to add the 0001 and keep the folder as `media`

call it multi-wallet-concept...

marcelruland commented:
> I did not saw it. @marcelruland please do not name the whitepaper as ssi eg wallet concepts!
> 
> Please remove the EG from the tractus-x things, also do not need to add the 0001 and keep the folder as `media`
> 
> call it multi-wallet-concept...

This repository is generally mixing Catena and Tractus. The white paper has been created by a Catena expert group, we cannot just remove that attribution. "Multi-wallet concept" does not adequately capture the contents.

We should keep IDs for white papers for the same reason we keep them for ADRs. I'm fine with renaming `img` to `media`.

matbmoser commented:
> > I did not saw it. @marcelruland please do not name the whitepaper as ssi eg wallet concepts!
> > Please remove the EG from the tractus-x things, also do not need to add the 0001 and keep the folder as `media`
> > call it multi-wallet-concept...
> 
> This repository is generally mixing Catena and Tractus. The white paper has been created by a Catena expert group, we cannot just remove that attribution. "Multi-wallet concept" does not adequately capture the contents.
> 
> We should keep IDs for white papers for the same reason we keep them for ADRs. I'm fine with renaming `img` to `media`.

General rule is not include Catena-X in Tractus-X, if is just some references its fine but now make it official like "expert group ssi". That is not the way.

matbmoser reviewed (changes requested):
I do not want to publish things from the expert group here, I can create a public repository in catena-x ev and there you can do it. Do not include the name SSI expert group. We are refactoring already all the Catena-X references on a high level at least, and making them tractus-x.

I will not approve until you add the correct naming.

docs/white-papers/byow-for-multi-wallet-providers.md

and the images don't add the `0001` also not in the document name. Here we dont want to have a chronology like in the decision records, this is just for decision records.

You can add a date in the document at a "metadata" level, and that would be enough.

marcelruland commented:
> I do not want to publish things from the expert group here, I can create a public repository in catena-x ev and there you can do it. Do not include the name SSI expert group (only some text references if you want). We are refactoring already all the Catena-X references on a high level at least, and making them tractus-x.

Then this white paper does not belong here. The recommendation to put it here came from you.

I agree that a TX repo is not the right place, but it's up to the Catena to come up with a suitable repo structure and decide where to put white papers like this one. I added a text reference to the SSI EG and removed it from the file name.

> I will not approve until you add the correct naming.
> 
> docs/white-papers/byow-for-multi-wallet-providers.md

This name does not reflect the contents of the document. I called it `multiple-wallet-providers-and-bring-your-own-wallet.md`.

> and the images don't add the `0001` also not in the document name. Here we dont want to have a chronology like in the decision records, this is just for decision records.

An ID would allow people to find more recent white papers more easily. I don't think removing it is a good idea. But also not a hill I'm willing to die on.

> The Eclipse Foundation projects have rules, and this points I am telling are enforced or should at least be inforced in all our Tractus-X repositories when it comes to documentation. This was told for us in the consortia time when we started to work in TX.
> 
> Please add the people that contributed, or the specific companies in the notice.

Copyright updated.

matbmoser commented on docs/white-papers/multiple-wallet-providers-and-bring-your-own-wallet/README.md:
```diff
@@ -0,0 +1,800 @@
+## Useful links
+
+* Related concept: [Moving to DID-based Identifiers in the Catena-X Protocol Layer](https://github.com/catenax-eV/cx-ex-dataspace-connectivity/blob/main/docs/bpn-did-resolution/moving-to-DID-identifiers.md#moving-to-did-based-identifiers-in-the-catena-x-protocol-layer)
+* Related issue: Deprecation StatusList2021, BitStringStatusList current W3C standard
```
Sorry for being so picky @marcelruland.

Add the license note here. 
Please use this template and add companies like we do in the KITs: 
![image](https://github.com/user-attachments/assets/d4e2cfb8-afab-4285-998a-a71d184a5e56)

```suggestion
* Related issue: Deprecation StatusList2021, BitStringStatusList current W3C standard

## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2025 Contributors to the Eclipse Foundation
- SPDX-FileCopyrightText: 2025 Bayerische Motoren Werke AG
- SPDX-FileCopyrightText: 2025 CGI Deutschland B.V. & Co. KG
- SPDX-FileCopyrightText: 2025 Cofinity-X GmbH
- SPDX-FileCopyrightText: 2025 DENSO Europe
- SPDX-FileCopyrightText: 2025 Metaform Systems Inc.
- Source URL: https://github.com/eclipse-tractusx/sig-architecture
```

marcelruland replied:
I've gone through a few iterations now and the requirements keep changing and evolving. I enabled edits by maintainers. Please amend as necessary. This is now by the way different from the [way it is done in the ADR template](https://github.com/eclipse-tractusx/sig-architecture/pull/10#discussion_r2000851851). Not sure if that was intended.

matbmoser replied:
then in the adr template is wrong... I will open a PR

matbmoser commented on docs/white-papers/multiple-wallet-providers-and-bring-your-own-wallet/media/compromised-issuer-key.png.license:
You can remove the AUTHORS, that was reserved for single persons, in case there was a specific person which made it, so it can be referenced. But it is optional.



```suggestion

## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

- SPDX-License-Identifier: CC-BY-4.0
- SPDX-FileCopyrightText: 2025 Contributors to the Eclipse Foundation
- SPDX-FileCopyrightText: 2025 Bayerische Motoren Werke AG
- SPDX-FileCopyrightText: 2025 CGI Deutschland B.V. & Co. KG
- SPDX-FileCopyrightText: 2025 Cofinity-X GmbH
- SPDX-FileCopyrightText: 2025 DENSO Europe
- SPDX-FileCopyrightText: 2025 Metaform Systems Inc.
- Source URL: https://github.com/eclipse-tractusx/sig-architecture
```

marcelruland replied:
See [my comment above](https://github.com/eclipse-tractusx/sig-architecture/pull/13#discussion_r2003189015).

matbmoser reviewed (changes requested):
Sorry for being so picky, but this is what we add in our TRG: https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08

We are also using it for the Repository Documentation. Specially for CC-BY-4.0

Once you add this on the licenses I can approve. Thank you @marcelruland for all the work and changes you have implemented :)

matbmoser reviewed (approved):
@marcelruland the only way I could commit was editing the files one by one for some reason, we will need to squash the commits so we dont overflow the commit history.

I understand your frustration, is my resposability as a committer to make sure the licenses are correctly added and that what we are adding is compliant to the open source guidelines the eclipse foundation gives us.

Now we should be fine for merging!

matbmoser commented:
please @evegufy and @lgblaumeiser review again and give your approval.
